### PR TITLE
Fuewa/lebenshilfe

### PR DESCRIPTION
**Pull request by Akira25**
This adds the Location at Lebenshilfe Oder-Spree in Fürstenwalde.

### CODE REVIEW

File: group_vars/location_fffw_lebenshilfe/networks.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  - vid: 40
+    role: dhcp
+    prefix: 10.30.99.0/25
+    #ipv6_subprefix: 0
+    assignments:
+      fffw-lebenshilfe-core: 1
+
+location__channel_assignments_11g_standard__to_merge:
+  # ap-name: Channel  Bandwidth  tx-Power
+  fffw-lebenshilfe-nno-ap-2ghz: 1-20-15
```
Comment (spolack) [inline group_vars/location_fffw_lebenshilfe/networks.yml:77]: is there a reason why you set txpower explicit?

File: host_vars/fffw-lebenshilfe-core/base.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+location: fffw-lebenshilfe
+role: corerouter
+target: "ipq40xx/generic"
```
Comment (spolack) [inline host_vars/fffw-lebenshilfe-core/base.yml:4]: its not necessary anymore to set the target per host. Nowadays this mapping went to model group_vars. e.g. https://github.com/freifunk-berlin/bbb-configs/blob/master/group_vars/model_avm_fritzbox_4040.yml#L5

.. It works by using 2 inventory stages. First one creating the group assignments based on host_vars, and a second one which can create group assignments of group_vars which were referenced in the first run.
https://github.com/freifunk-berlin/bbb-configs/tree/master/inventory

File: host_vars/fffw-lebenshilfe-core/base.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+model: "avm_fritzbox-7530"
+
+wireless_profile: freifunk_fw
```
Comment (spolack) [inline host_vars/fffw-lebenshilfe-core/base.yml:7]: you could consider setting the wireless_profile within your location group_vars. saving some redundant configuration.

we can also think of introducing a group per community. therefore you could override the wireless_profile for all hosts belonging to a community.

File: host_vars/fffw-lebenshilfe-nno-ap-2ghz/base.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+location: fffw-lebenshilfe
+role: ap
+target: "ath79/generic"
+model: "mikrotik_sxtsq-2-lite"
+
+wireless_profile: freifunk_fw
+
+mac_override:
+  eth0: cc:2d:e0:9c:4d:58
+...
```
Comment (spolack) [inline host_vars/fffw-lebenshilfe-nno-ap-2ghz/base.yml:11]: ?

**Review by spolack — CHANGES_REQUESTED** [review]

File: group_vars/location_fffw_lebenshilfe/networks.yml (hunk 1 above)
Comment (Akira25) [reply group_vars/location_fffw_lebenshilfe/networks.yml:77]: No, there isn't. i just copied that from the documentation, which was derived from kiehlufer-core, I think... Can I omit the tx-power and it gets set automatically then?

File: host_vars/fffw-lebenshilfe-core/base.yml (hunk 2 above)
Comment (Akira25) [reply host_vars/fffw-lebenshilfe-core/base.yml:4]: Good to know. Hopefully I will get some time and adjust the examples in the documentation accordingly.

File: host_vars/fffw-lebenshilfe-core/base.yml (hunk 3 above)
Comment (Akira25) [reply host_vars/fffw-lebenshilfe-core/base.yml:7]: The latter solution sounds really attactive to me. For the sake of getting the site online fast, I'd go for the first solution for now. :)

File: host_vars/fffw-lebenshilfe-nno-ap-2ghz/base.yml (hunk 4 above)
Comment (Akira25) [reply host_vars/fffw-lebenshilfe-nno-ap-2ghz/base.yml:11]: I've saw that configuration on a couple of locations using the Mikrotik-SXTsq5ac-antennas. I thought that it needs to be that way. Does OpenWrt get the correct MAC-Address on those devices by itself?

Comment (spolack) [reply host_vars/fffw-lebenshilfe-nno-ap-2ghz/base.yml:11]: No the comment is because of L11

Comment (Akira25) [reply host_vars/fffw-lebenshilfe-nno-ap-2ghz/base.yml:11]: oh, that one. I 've read somewhere that it's not strictly necessary, but good practice to end a YAML-File with three dots. Like it was begun with three dashes.

**Comment by spolack** [comment]
never seen that before, but you might be right.

File: group_vars/location_fffw_lebenshilfe/networks.yml (hunk 1 above)
Comment (Akira25) [reply group_vars/location_fffw_lebenshilfe/networks.yml]: tx-power removed in new version.

File: host_vars/fffw-lebenshilfe-core/base.yml (hunk 2 above)
Comment (Akira25) [reply host_vars/fffw-lebenshilfe-core/base.yml]: target in host-vars removed.

File: host_vars/fffw-lebenshilfe-core/base.yml (hunk 3 above)
Comment (Akira25) [reply host_vars/fffw-lebenshilfe-core/base.yml:6]: As I had difficulties at defining the wifi profile in group-vars, I left them in host-vars for now.

File: host_vars/fffw-lebenshilfe-nno-ap-2ghz/base.yml (hunk 4 above)
Comment (Akira25) [reply host_vars/fffw-lebenshilfe-nno-ap-2ghz/base.yml:10]: I think that one has been resolved.

**Review by spolack — APPROVED** [review]

**Comment by Akira25** [comment]
> Still you could consider basing off `add_tunnelmanager` and enable IPv6. Just choose some District, the routing really doesnt care. And wont in the future, until we restructure IPv6 addressing from scratch

I think, we will postpone IPv6 a bit in Fürstenwalde. For IPv4, we've allocated our own block in the intercity-vpn repository. We didn't speak on IPv6 for, now, but I will ask the other community members of fuerstenwalde what they like to do.

Anyway I think we will add that then. :)

**Pull request merged**